lib/components/post-editor-toolbar-component.js

```diff
@@ -15,24 +15,21 @@ export default class PostEditorToolbarComponent extends BaseContainer {
 		this.publishButtonSelector = By.css( '.editor-publish-button' );
 	}
 
-	ensureSaved() {
+	ensureSaved( clickSave = true ) {
 		const saveSelector = By.css( 'button.editor-ground-control__save' );
-		const savingSelector = By.css( 'span.editor-ground-control__saving' );
+		const savedSelector = By.css( 'span.editor-ground-control__save-status[data-e2e-status="Saved"]' );
 		const driver = this.driver;
 
-		driverHelper.clickIfPresent( driver, saveSelector, 3 );
-
-		driver.wait( function() {
-			return driverHelper.isElementPresent( driver, saveSelector ).then( function( present ) {
-				return ! present;
-			} );
-		}, this.explicitWaitMS, 'The save selector was still present when it should have disappeared with auto-save.' );
+		if ( clickSave === true ) {
+			driverHelper.clickIfPresent( driver, saveSelector, 3 );
+		}
 
 		return driver.wait( function() {
-			return driverHelper.isElementPresent( driver, savingSelector ).then( function( present ) {
-				return ! present;
+			return driverHelper.isElementPresent( driver, savedSelector ).then( function( present ) {
```

Comment:
I think this whole block could be replaced with calling `driverHelper.waitTillPresentAndDisplayed(...)` (I think this existing code pre-dates that helper method)